src/v3/src/util/formUtils.ts

```diff
@@ -492,3 +525,144 @@ export const buildPhoneVerificationSubtitleElement = (
 
   return subtitleElement;
 };
+
+type RequestOptions = {
+  url: string;
+  timeout: number;
+  method: 'GET' | 'POST';
+  data?: string;
+};
+
```

Comment:
I think there's enough loopback-specific logic here to warrant a whole separate utility file outside of `formUtils`